test/WebJobs.Extensions.DurableTask.Analyzers.Test/ActivityFunction/FunctionReturnTypeAnalyzerTests.cs

```diff
@@ -100,14 +104,20 @@ public static Object SayHello([ActivityTrigger] IDurableActivityContext context)
             return tuple;
         }
 
+        [FunctionName(""E1_SayHello_ValueTuple"")]
+        public static ValueTuple<string, int> SayHelloTuple([ActivityTrigger] ValueTuple<string, int> tuple)
+        {
+            return tuple;
+        }
+
         [FunctionName(""E1_SayHello_ReturnsString"")]
         public static string SayHelloDirectInput([ActivityTrigger] string name)
         {
             return $""Hello {name}!"";
         }
 
         [FunctionName(""E1_SayHello_ArrayAndNamedType"")]
```

Comment:
Not sure I understand what `ArrayAndNamedType` means exactly. I was able to look up that `INamedTypeSymbol` is any type other than "an array, a pointer, a type parameter", but that isn't immediately obvious what this function is trying to accomplish. Is the idea that ValueTuple is a NamedType, and List<string>() is an array type? If that is the case, then I would make the name more descriptive, like "E1_SayHello_InputNamedType_ReturnsArray" or something along those lines.

Reply:
For the types that are serialized to JsonArrays (implement IEnumerable) there are IArrayTypeSymbols and INamedTypeSymbols. This test is comparing the two here. I agree, it would be a good idea to rename it.